src/adapters/PressureSensorAdapter.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * CS 410 - Assignment 2 
+ * @author Daniyal Adzhiyev
+ * @version 1.0 05/28/2019
+ * 
+ * This Application uses the Adapter Software Pattern to create adapter's for multiple predefined sensor classes.
+ * Where the adapter classes implement the Sensor Application Interface, to 'adapt' the original sensor classes
+ * so that all of the adapter classes have the same methods needed to display the information in the Sensor 
+ * Application.
+ *  
+ */
+
+package adapters;
+
+import java.awt.Color;
+
+import sensor.PressureSensor;
+
+/*
+ * This class is an adapter for the Pressure Sensor class to allow communication between the Sensor Application, to adapt
+ * the methods from the Pressure Sensor, so that it fits the needs of the Sensor Application.
+ */
+public class PressureSensorAdapter implements SensorApplicationInterface {
+	PressureSensor adaptee; // creates an adaptee object to allow communication to the Pressure Sensor class
+	private double value; // value of the Pressure Sensor reading
```

Comment:
Probably don't need this. The Adapter class only needs to adapt the methods in PressureSensor, not the variables. And if the variable is needed, there is a get method for it.